client/src/pages/Discover/WinnersCard/Winner.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import useStyles from './useStyles';
+import moment from 'moment';
+import { Typography, Grid, Card, CardActionArea, CardMedia } from '@material-ui/core';
+
+interface Props {
+  image: string;
+  designTitle: string;
+  contestTitle: string;
+  username: string;
+  winningDate: string;
+}
+
+export default function WinnerCard({ image, designTitle, contestTitle, username, winningDate }: Props): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Card elevation={0} className={classes.cardBackground}>
+      <Grid container>
+        <Grid item xs={12}>
+          <Typography variant="body1">{contestTitle}</Typography>
+        </Grid>
+        <Grid item xs={12}>
+          <Typography variant="body1">{moment(winningDate).format('MMM Do, YYYY')}</Typography>
+        </Grid>
+      </Grid>
+      <CardActionArea>
+        <CardMedia>
```

Comment:
Same as comment above for Card Media component

Reply:
Successfully implemented at commit 03e97656998da9be9788a9e83a0d734c45c19266